array.c

```diff
@@ -4636,6 +4636,10 @@ rb_ary_difference_multi(int argc, VALUE *argv, VALUE ary)
 static VALUE
 rb_ary_and(VALUE ary1, VALUE ary2)
 {
+    // VALUE arg[1];
+    // arg[0] = ary2;
+    // return rb_ary_intersection_multi(1, arg, ary1);
```

Comment:
For some reason, when I try to rewrite `&` in terms of `intersection`, I'm getting a weird error:
```
Traceback (most recent call last):
	7: from ./tool/transform_mjit_header.rb:5:in `<main>'
	6: from ./tool/transform_mjit_header.rb:5:in `require'
	5: from /Users/travis/code/ruby/lib/fileutils.rb:106:in `<top (required)>'
	4: from /Users/travis/code/ruby/lib/fileutils.rb:1627:in `<module:FileUtils>'
	3: from /Users/travis/code/ruby/lib/fileutils.rb:1627:in `inject'
	2: from /Users/travis/code/ruby/lib/fileutils.rb:1627:in `each'
	1: from /Users/travis/code/ruby/lib/fileutils.rb:1628:in `block in <module:FileUtils>'
/Users/travis/code/ruby/lib/fileutils.rb:1628:in `instance_method': undefined method `private_module_function' for module `FileUtils' (NameError)
```